### ISSUE TITLE
No prediction result !

### ISSUE THREAD

**Issue by cwnuyangyan** [WKWiecas/IOT_OIE#3]
I reproduce the demo, **use imojie to train, use carb corups to predict the result**, I run the detie_predict.py, I can't get the predict result; 
I check the demo, in allpy.py, the function  **selection_of_triples_with_argmax** can't give any detections. So could you provide the pretained model ? or Could you provide the detailed train process ?
![image](https://github.com/WKWiecas/IOT_OIE/assets/39004305/f868ec4d-dc6d-4674-bb07-f256cd484b94)
![image](https://github.com/WKWiecas/IOT_OIE/assets/39004305/cc948731-9fc5-4a4b-ac07-cb0c805a9297)

this is my params:
`model_cfg:
  join_is: true
  seed: 100
  profile: false
  gpus: 1
  log_every_n_steps: 100
  validate_on_test: false
  val_fraction: 0.1
  n_test_batches: 100
  test_results_file: results/model_test.json
  tokenizer: ./bert-base-multilingual-cased
  best_ckpt: best.ckpt
  best_hparams_path: results/logs/default/version_3/hparams.yaml
  best_ckpt_path: results/logs/default/version_3/checkpoints/best.ckpt
  cache_dir: results/cache
  viz_sentences:
  - This is how Agent Archer received a medal from the US.
  - The assault rifle is one of the very popular weapons in the Sahara desert.
  - Bill Gates owns Microsoft Corporation.
  - Fallout 4 is a post-apocalyptic role-playing game developed by Bethesda Game Studios[2]
    and published by Bethesda Softworks.
  - A new equipment system allows a wide variety of upgrades and customization for
    weapons and armor.
  - Each sample will be retrieved by indexing tensors along the first dimension.
  - Skills have been removed, replaced with a 7x10 perk chart. At each level-up, the
    player can acquire a new perk, each with prerequisites based on SPECIAL attributes
    and level, and most have multiple tiers.
  - The Wells and Wellington affair was a dispute involving the Australian Journal
    of Herpetology, a scientific journal on the study of amphibians and reptiles published
    beginning in 1981 by the Australian Herpetologists' League. Richard Wells, a student,
    served as the editor-in-chief of the peer-reviewed periodical, with an editorial
    board of three researchers.
  name: TripletsExtractorBERTOnly
  lang: en
  use_syntax_features: false
  postprocess_adp: false
  pretrained_encoder: ./bert-base-multilingual-cased
  word_dropout: 0
  stanza_emb_size: 32
  max_epochs: 100
  batch_size: 64
  unfreeze_epoch: 0
  unfreeze_layers_from_top: 4
  syntetic_data_after_epoch: 100
  matching: iou
  disable_bg: true
  focal_gamma: 2
  class_weights:
  - 1
  - 1
  - 1
  - 1
  num_layers: 2
  hid_size: 512
  n_classes: 4
  num_detections: 20
  best_version: 3
  top_candidates: 1
  dynamic_k: true
  ot_iteration: 50
  iou_ceil: true
  topk_max: 3
  beta: 1
  mu: 1
  use_focal: false
  use_dice: false
  use_reweight: false
  logit_method: 0
opt_cfg:
  name: Adam
  lr: 1.0e-05
  weight_decay: 1.0e-06
scheduler_cfg:
  name: ExponentialLR
  gamma: 1
`
